src/redux/books/books.js

```diff
@@ -1,46 +1,42 @@
-import { addBookApi, getBookApi, removeBookApi } from '../../Api/booksApi';
+import { apiAddBook, apiGetBooks, apiRemoveBook } from '../../modules/api';
 
-const ADD_BOOK = 'bookstore/books/ADD_BOOK';
-const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
-const GET_BOOK = 'bookstore/books/GET_BOOK';
-
-const initialState = [];
+const ADD = 'bookstore/Book/ADD';
+const GET = 'bookstore/Book/GET';
+const REMOVE = 'bookstore/Book/REMOVE';
 
 export const addBook = (payload) => async (dispatch) => {
-  const book = { ...payload, item_id: payload.id };
-  await addBookApi(book);
-  dispatch({ type: ADD_BOOK, book });
-};
-
-export const removeBook = (id) => async (dispatch) => {
-  await removeBookApi(id);
-  dispatch({ type: REMOVE_BOOK, id });
+  const book = { ...payload, item_id: payload.id, category: 'category' };
+  await apiAddBook(book);
+  dispatch({ type: ADD, book });
 };
 
-export const getBook = () => async (dispatch) => {
-  const data = await getBookApi();
-  const books = Object.keys(data).map((id) => {
-    const book = data[id][0];
-    book.id = id;
+export const getBooks = () => async (dispatch) => {
+  const data = await apiGetBooks();
+  const books = Object.keys(data).map((key) => {
+    const book = data[key][0];
+    book.id = key;
     return book;
   });
-  dispatch({ type: GET_BOOK, books });
+  dispatch({ type: GET, books });
+};
+
+export const removeBook = (id) => async (dispatch) => {
+  await apiRemoveBook(id);
+  dispatch({ type: REMOVE, id });
```

Comment:
- [ ] all your actions should look similar to this using `createAsyncThunk` to return a promise
```
export const fetchBooks = createAsyncThunk("books/fetchBooks", async () => {
  const response = await axios.get(
    "https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/"
  )
  return response.data
})
```